api/download.php

```diff
@@ -22,40 +22,49 @@ function go_home() {
     global $sitewide;
 
     header("Location: " . $sitewide['root']);
-    return;
+    die();
 }
 
 // everything else falls into a great pyrimid of php ifs
 $charge_id = $_GET['charge'];
 
 if (substr($charge_id, 0, 3) !== 'ch_') {
-    go_home();
+    return go_home();
 }
 
 try {
     $charge = \Stripe\Charge::retrieve($charge_id);
 } catch (Exception $e) {
-    go_home();
+    return go_home();
 }
 
 $products = array();
 if (isset($charge['metadata']['products'])) {
     try {
         $products = json_decode($charge['metadata']['products']);
     } catch (Exception $e) {
-        go_home();
+        return go_home();
     }
 }
 
-$iso_version = false;
+$isoVersion = false;
 foreach ($products as $product) {
     if (substr($product, 0, 3) === 'ISO') {
-        $iso_version = substr($product, 4);
+        $isoVersion = substr($product, 4);
```

Comment:
Is this literally taking the first 4 characters? That won't work with double digits, why not just handle them with explode like you do later?